packages/spatial/src/renderer/WebGLRendererSystem.tsx

```diff
@@ -213,9 +214,59 @@ export const RendererComponent = defineComponent({
 
     useEffect(() => {
       const canvas = rendererComponent.canvas.value as HTMLCanvasElement
+
+      const handleWebGLContextLost = (e) => {
```

Comment:
we should test context loss again

Reply:
context restore is still not working. We should leave that for a future PR. Needs to be investigated further.